samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -235,6 +235,16 @@ static constexpr char inference_only_message[] =
     " To enable full mode for static models pass \"false\" value to this argument:"
     " ex. \"-inference_only=false\".\n";
 
+// @brief message for load property option
+static const char load_property_message[] =
+    "Optional. Path to JSON file to load custom property for OpenVINO runtime API 2.0."
+    " Please note, command line parameters have higher priority than parameters from configuration "
+    "file.";
+
+// @brief message for dump property option
+static const char dump_property_message[] = "Optional. Path to JSON file to dump properties for OpenVINO runtime API "
```

Comment:
I suppose it is very confusing to have 2 different set of config options

Reply:
Actually, one is for OV 2.0 API/interface and another is for legacy API/interface, there are different. In current benchmark, only legacy API/interface config can be loaded and dumped, but OV 2.0 API's property is not supported to dumped and loaded.
This PR is to support OV 2.0 API's property dumped and loaded.